### PR TITLE
add brier score to default classification metrics

### PR DESCRIPTION
**Pull request by topepo**
There causes some extratests breakages but those are pretty easy to change.

### CODE REVIEW

**Comment by topepo** [comment]
I did not see any revdep failures besides the existing ones.

File: revdep/problems.md [hunk 1]
```diff
@@ -1 +1,146 @@
...
+       3.   └─tune:::new_bare_tibble(...)
+       4.     └─tibble::new_tibble(x, nrow = nrow(x), ..., class = class)
+       5.       └─rlang::pairlist2(...)
+      
+      [ FAIL 1 | WARN 0 | SKIP 20 | PASS 107 ]
+      Error: Test failures
+      Execution halted
+    ```
+
+# tidyclust
```
Comment (topepo) [inline revdep/problems.md:42]: @EmilHvitfeldt

**Review by simonpcouch — COMMENTED** [review]
Happy to see that this looks pretty straightforward so far!

Many of the tests in tune use the example generated objects in `test/testthat/data`. Could we rerun `inst/test_objects.R` and see how tests look after that?

**Comment by topepo** [comment]
@simonpcouch CI is good and no additional revdep issues

**Review by simonpcouch — APPROVED** [review]
Awesome, thanks for rerunning those. Let's do it!🙂

Not related to this PR specifically, but noting that we now have a circular dependency with tune and finetune. (tune needs a test change for finetune, finetune needs dev tune features.)

**Pull request merged**